### ISSUE TITLE
Is this still a better choice than golang roc? 

### ISSUE THREAD

**Issue by 1a1a11a** [valyala/gorpc#33]
I see the last commit is more than two years ago, so I would like to know whether this is better choice if I want to use it in high-load servers with large request size (~200K content per request on average).

**Comment by MostHated** [comment]
I was really wanting to like this library and use it, but it just wasn't happening unfortunately. I just kept having a bunch of issues. I am not sure if it is helpful anymore (or even in your particular case) but, I decided to [give rpcx a go](https://github.com/smallnest/rpcx) and I am thrilled with it so far. 

I spent two days trying to get this library working how I needed, I was able to swap it out for rpcx in about an hour and a half and everything is working perfect. I was pleasantly surprised when I saw it uses MessagePack, too. I am a big fan of that library from using it in Unity/C# for networking.
